src/demo/app.py

```diff
@@ -75,9 +74,9 @@ def call_caption_api(inpainted_image):
     raise Exception(f"Error: {response.status_code} - {response.text}")
 
 
-def call_procap_api(caption):
+def call_procap_api(caption: str):
     """
-    Classifies the caption as harmful or not.
+    Classifies the meme as harmful or not based on the caption.
```

Comment:
```suggestion
    Classifies the meme as harmful or not based on the caption.
    Classifies the caption as harmful or not.
```
there isnt any meme context involved in this api thus we shouldlnt mention it